lib/hammer_cli_katello/ping.rb

```diff
@@ -6,24 +6,32 @@ class PingCommand < HammerCLIKatello::Command
 
     output do
       from "services" do
+        label "katello_agent", hide_blank: true do
```

Comment:
Future versions of Katello won't have katello_agent enabled (will be missing in ping API) - can we handle that now?